pyproject.toml

```diff
@@ -13,10 +13,15 @@ requires = [
     "pywinpty (>=0.5);os_name=='nt'",
     "tornado (>=4)",
 ]
+python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*"
```

Comment:
My take is that this is more trouble than it's worth until we can express it concisely (`>=3.4`). But I'm open to other opinions.

The field in flit is called `requires-python`, though: http://flit.readthedocs.io/en/latest/pyproject_toml.html#metadata-section

Reply:
Thanks, I've removed the x.y classifiers, and corrected this field.